mk/spksrc.wheel.mk

```diff
@@ -44,34 +44,32 @@ pre_wheel_target: wheel_msg_target
 			mkdir -p $(PIP_DIR) ; \
 		fi; \
 		mkdir -p $(WORK_DIR)/wheelhouse ; \
-		if [ -f "$(WHEELS)" ] ; then \
-			$(MSG) "Using existing requirements file" ; \
-			cp -f $(WHEELS) $(WORK_DIR)/wheelhouse/requirements.txt ; \
-		else \
-			$(MSG) "Creating requirements file" ; \
-			rm -f $(WORK_DIR)/wheelhouse/requirements.txt ; \
-			for wheel in $(WHEELS) ; \
-			do \
+		for wheel in $(WHEELS) ; \
+		do \
+			if [ -f $$wheel ] ; then \
+				$(MSG) "Using existing requirements.txt file" ; \
+				cp -f $$wheel $(WORK_DIR)/wheelhouse/$$(basename $$wheel) ; \
+			else \
+				$(MSG) "Adding to requirements.txt file" ; \
 				echo $$wheel >> $(WORK_DIR)/wheelhouse/requirements.txt ; \
-			done \
-		fi ; \
+			fi ; \
+		done \
 	fi
 
 build_wheel_target: $(PRE_WHEEL_TARGET)
 	@if [ ! -z "$(WHEELS)" ] ; then \
 		$(foreach e,$(shell cat $(WORK_DIR)/python-cc.mk),$(eval $(e))) \
-		if [ ! -z "$(CROSS_COMPILE_WHEELS)" ] ; then \
+		if [ -f "$(WORK_DIR)/wheelhouse/cross-requirements.txt" ]; then \
```

Comment:
So what happens here? How does the rename from `requirements` to `cross-requirements` fixes things?

Reply:
its not only a rename but rather telling to cross-compile the ones in `cross` and "python" compile the other ones by default (e.g. `pure`).  Again for reference https://github.com/SynoCommunity/spksrc/issues/4925#issuecomment-946304515